src/frontend/js/pages/DashboardOpenEdxProfile/index.tsx

```diff
@@ -225,13 +231,9 @@ const DashboardOpenEdxProfile = () => {
       </DashboardBox.List>
 
       <Form.Footer>
-        <RouterButton
-          fullWidth
-          target="_blank"
-          href={`${context.authentication.endpoint}/account/settings`}
-        >
+        <Button fullWidth onClick={onClickModifyProfile}>
           <FormattedMessage {...messages.editButtonLabel} />
-        </RouterButton>
+        </Button>
```

Comment:
For a11y purpose, I think we could use href attribute to generate a link.
Does following suggestion work ?

```suggestion
        <Button
          fullWidth
          href={`${context.authentication.endpoint}/account/settings`}
          onClick={clearReactQueryCache}
        >
          <FormattedMessage {...messages.editButtonLabel} />
        </Button>
```